src/controller/AutoCommissioner.h

```diff
@@ -38,9 +38,22 @@ class AutoCommissioner : public CommissioningDelegate
 
     CHIP_ERROR CommissioningStepFinished(CHIP_ERROR err, CommissioningDelegate::CommissioningReport report) override;
 
+    /**
+     * @brief
+     *   This function puts the AutoCommissioner in a paused state to prevent advancing to the next stage.
+     * It is expected that a DevicePairingDelegate may call this method when processing the
+     * OnCommissioningStatusUpdate, for example, in order to obtain network credentials from the user based
+     * upon the results of the NetworkScan.
+     * Use ResumeCommissioning to continue the commissioning process.
+     *
+     */
+    void PauseCommissioning();
+    CHIP_ERROR ResumeCommissioning();
```

Comment:
Should document what an error return from here means.

Reply:
done